### DISCUSSION TITLE
İlk listelendiğinde fiyat ne olur sizce ?

### DISCUSSION THREAD

**Discussion by cihanaslan0** [IBAX-io/go-ibax#109] [Q&A]
Tatmin edici olur mu sizce ?

**Comment by Emredurmus27** [comment]
Bilemez kimse malesef

**Reply by cihanaslan0** [reply]
Tahmin sadece 😄

**Comment by cartvecurt** [comment]
İlk listelemede yüksek olacağını hiç sanmıyorum, benzer arzlı projelerden bakabilirsin belki

**Comment by hemresenol** [comment]
0,5 cent olur

**Reply by cihanaslan0** [reply]
En aşağı 1 dolar olur söyleyeyim

**Comment by Skips1** [comment]
Piyasaya bağlı biraz kanka :)
